tests/unit/test_parser.py

```diff
@@ -37,3 +48,27 @@ def test_incorrect_banner_ios():
     )
     with pytest.raises(ValueError):
         compliance.parser_map["cisco_ios"](banner_cfg).config_lines  # pylint: disable=expression-not-assigned
+
+
+@pytest.mark.parametrize("_file, pattern, expected", get_path_parameters)
```

Comment:
ok, I see you added it here, but didn't define `get_path_parameters`, in #141 you should see how that is auto populated based on folder structure. The premise is you just put files in specfic locations, and it will dynamically find it

Reply:
I still do not understand what you are asking. This is a set of parameters for the unit test. Did you want to dynamically populate it for different platforms? You can do this with parametrize.

Reply:
I took this from 141, this uses glob to dynamically find files, which populates the variables for paramatized variable. With these two concepts you can create a framework, in which you simply drop files in a certain location, and they are automatically tests.

```
base_parameters = []
find_all_children_parameters = []
find_children_w_parents_parameters = []
for network_os in list(compliance.parser_map.keys()):
    for _file in glob.glob(f"{MOCK_DIR}/base/{network_os}/*{TXT_FILE}"):
        base_parameters.append([_file, network_os])
    for _file in glob.glob(f"{MOCK_DIR}/find_all_children/{network_os}/*{TXT_FILE}"):
        find_all_children_parameters.append([_file, network_os])
    for _file in glob.glob(f"{MOCK_DIR}/find_children_w_parents/{network_os}/*{TXT_FILE}"):
        find_children_w_parents_parameters.append([_file, network_os])
```

Reply:
Addressed.